index.js

```diff
@@ -1,52 +1,19 @@
 /* jshint node: true */
 'use strict';
 
-var defaults = {
-  css: true,
-  javascript: true,
-  fonts: true,
-  images: true
-};
-
-var getWithDefault = function(property, default_property) {
-  if (property === null || property === undefined) {
-    return default_property;
-  }
-
-  return property;
-};
+var buildPaths = require('./utils/paths');
 
 module.exports = {
   name: 'semantic-ui-ember',
 
   included: function (app) {
     var options = (app && app.options['SemanticUI']) || {};
-
-    if (getWithDefault(options['css'], defaults['css'])) {
-      app.import({
-        development: 'bower_components/semantic-ui/dist/semantic.css',
-        production: 'bower_components/semantic-ui/dist/semantic.min.css'
-      });
-    }
-
-    if (getWithDefault(options['javascript'], defaults['javascript'])) {
-      app.import({
-        development: 'bower_components/semantic-ui/dist/semantic.js',
-        production: 'bower_components/semantic-ui/dist/semantic.min.js'
-      });
-    }
-
-    if (getWithDefault(options['images'], defaults['images'])) {
-      var imageOptions = { destDir: 'assets/themes/default/assets/images' };
-      app.import('bower_components/semantic-ui/dist/themes/default/assets/images/flags.png', imageOptions);
-    }
-
-    if (getWithDefault(options['fonts'], defaults['fonts'])) {
-      var fontExtensions = ['.eot','.otf','.svg','.ttf','.woff','.woff2'];
-      var fontOptions = { destDir: 'assets/themes/default/assets/fonts' };
-      for (var i = fontExtensions.length - 1; i >= 0; i--) {
-        app.import('bower_components/semantic-ui/dist/themes/default/assets/fonts/icons'+fontExtensions[i], fontOptions);
-      };
-    }
+    buildPaths(options).forEach(function(path) {
+      if (Array.isArray(path)) {
+        app.import(path[0], path[1]);
```

Comment:
I've never used import like that before. I thought it was always a string or an object. I couldn't see if in the documentation either. Can you point me to where this is supported instead of an object?